config/stage.json

```diff
@@ -31,6 +31,7 @@
 		"manage/ads/jetpack": true,
 		"manage/ads/wordads-instant": true,
 		"manage/advanced-seo": true,
+		"manage/advanced-seo/custom-title": true,
```

Comment:
I would probably avoid this until you're ready to release this into prod. Maybe enable this on wpcalypso or horizon instead?